### PR TITLE
Release 0.5.0 docs 

### PR DESCRIPTION
**Pull request by tsande16**
Release actions checklist for 0.5.0 updated
New checklist for 0.6.0 created
Updated Release Manager Checklist template with new tasks/visually rearranging checklist.

Release notes for 0.5.0 created. The Release label for 0.5.0 was added to all done tasks on the Kanban board as well. Draft for google groups sent to Mark & John K.

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+## Release v0.5.0
+### Date: April 27, 2023
+This release introduces the Metadata Schema Service and the Policy Service API. Release Automations have been expanded to include Pass-Acceptance-Testing and Pass-Docker.
```
Comment (bbranan) [inline docs/release-notes.md:3]:
```suggestion
This release introduces the Metadata Schema Service and the Policy Service API. Release Automation has been expanded to include [pass-acceptance-testing](https://github.com/eclipse-pass/pass-acceptance-testing) and [pass-docker](https://github.com/eclipse-pass/pass-docker).
```

File: docs/release-notes.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+
+* More release automations added to include Pass-Acceptance-Testing and Pass-Docker
```
Comment (bbranan) [inline docs/release-notes.md:5]: These three bullets really just repeat the sentence above, so aren't that helpful.

File: docs/release-notes.md (hunk 1 above)
Comment (bbranan) [reply docs/release-notes.md:3]: A brief (few word) description of the purpose of the Metadata Schema Service and Policy Service would help make this more understandable for those not already familiar with how PASS works.

File: docs/release/release-actions-0.6.0.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+# Release Manager Actions Checklist 0.6.0
```
Comment (bbranan) [inline docs/release/release-actions-0.6.0.md:1]: I'm not sure it's necessary to create this page for the next release now, given that it won't be used for a while. I'd be inclined to let the release manager for the next release create this from the template as a first step in the release procedure.

**Review by bbranan — CHANGES_REQUESTED** [review]
This looks good. I suggested a few minor adjustments.

File: docs/release/release-actions-0.6.0.md (hunk 3 above)
Comment (tsande16) [reply docs/release/release-actions-0.6.0.md:1]: Sounds good. I will delete this.

File: docs/release/release-actions-template.md [hunk 4]
```diff
@@ -15,28 +15,40 @@ This is the full detailed release process, including the steps that are performe
...
+Note: This must be released last because it relies on some Docker images that will be published during the release process.
 
-- [ ] Release Pass Docker - [Release workflow](https://github.com/eclipse-pass/pass-docker/actions/workflows/release.yml)
+- [ ] Release Pass Docker - Select checkbox for acceptance tests - [Release workflow](https://github.com/eclipse-pass/pass-docker/actions/workflows/release.yml)
+- [ ] Pass Docker Release Page - Perform after the Pass Docker release is complete - [Pass Docker GitHub Release Page](https://github.com/eclipse-pass/pass-docker/releases)
 
 ## Post-release
 
 - [ ] Test the release by using the newly updated pass-docker to run the release locally.
+- [ ] Create label for release. Add labels to Issues in Eclipse-Pass/Main (Kanban Board) that are associated with the release.
```
Comment (jabrah) [inline docs/release/release-actions-template.md:51]: Should add a link to the kanban board (https://github.com/orgs/eclipse-pass/projects/4/views/2) for convenience

**Review by jabrah — COMMENTED** [review]
Seems fine to me overall. Left one comment on the release action template

**Review by jabrah — APPROVED** [review]

**Review by markpatton — APPROVED** [review]
Looks good.

**Pull request merged**